cmd/versitygw/admin.go

```diff
@@ -289,11 +289,15 @@ func listUsers(ctx *cli.Context) error {
 	}
 	defer resp.Body.Close()
 
+	if resp.StatusCode >= 400 {
+		fmt.Printf("%s", body)
+		return nil
```

Comment:
dont we want to return an error here so that the program exits with an error status?